Resources/text/fr/lang_fr.txt

```diff
@@ -18,41 +18,41 @@
 ---------------------Menu Strip------------------------
 Menu_File = File
 Menu_Open = Ouvrir...
-Menu_Save = Enregistrer PK6...
-Menu_ExportSAV = Enregistrer SAV...
-Menu_ExportMAIN = Enregistrer main
+Menu_Save = Enregistrer en fichier PK6...
+Menu_ExportSAV = Enregistrer en fichier SAV...
+Menu_ExportMAIN = Enregistrer Main
 Menu_ExportBAK = Enregistrer BAK
 Menu_Exit = Quitter
 --
 Menu_Tools = Outils
 Menu_Showdown = Showdown
-Menu_ShowdownImportPK6 = Import Set from Clipboard
-Menu_ShowdownExportPK6 = Export Set to Clipboard
-Menu_ShowdownExportParty = Export Party to Clipboard
-Menu_ShowdownExportBattleBox = Export Battle Box to Clipboard
+Menu_ShowdownImportPK6 = Importer le set du presse-papier
+Menu_ShowdownExportPK6 = Exporter le set dans le presse-papier
+Menu_ShowdownExportParty = Exporter les pokémons en votre possession dans le presse-papier
+Menu_ShowdownExportBattleBox = Exporter la Boîte Combat dans le presse-papier
 Menu_CyberGadget = CyberGadget
-Menu_OpenTemp = Ouvrir le dossier Temp
+Menu_OpenTemp = Ouvrir le dossier Temporaire
 Menu_OpenCache = Ouvrir le dossier Cache
 Menu_Data = Données
-Menu_LoadBoxes = Charger Boîtes
-Menu_DumpBoxes = Dumper Boîtes
+Menu_LoadBoxes = Charger les Boîtes
+Menu_DumpBoxes = Jeter les Boîtes
```

Comment:
Dumper (Dump) shall be translate as "Sauvegarder/Extraire" instead of "Jeter" since it stands for "Drop/Throw".